tests/tendermint/test_lib.py

```diff
@@ -176,54 +183,6 @@ def test_update_utxoset(tb, signed_create_tx, signed_transfer_tx, db_context):
     assert utxo['output_index'] == 0
 
 
-@pytest.mark.bdb
-def test_store_transaction(mocker, tb, signed_create_tx,
```

Comment:
Why was this test removed?

Reply:
Because we use the `store_bulk_transactions` now. I can put it in again and switch the `store_transaction` to `store_transactions` if you want

Reply:
@codegeschrei This test is also trying to assert if the transaction was appropriately split and the metadata and asset were stored in their respective collections. It would good to have this test back.